performance/patient-create-stress.js

```diff
@@ -0,0 +1,42 @@
+import http from 'k6/http'
+import { check } from 'k6'
+
+const patientResource = require('./resources/patient.js')
+
+const patient = JSON.stringify(patientResource)
+
+/* global __ENV */
+const BASE_URL = __ENV.BASE_URL || 'http://localhost:3447'
+
+// PATH - the path to the resource. Default is path to the patient resource
+const RESOURCE_PATH = __ENV.RESOURCE_PATH || '/fhir/Patient'
+
+export const options = {
+  vus: 100,
+  duration: '2m',
+  thresholds: {
+    http_req_duration: ['p(95)<60']
+  },
+  noVUConnectionReuse: true,
+  discardResponseBodies: true
+}
+
+const makePostRequest = () => {
+  const response = http.post(`${BASE_URL}${RESOURCE_PATH}`,
+    patient,
+    {
+      headers: {
+        Accept: 'application/json'
+      },
+      tags: {
+        name: 'Post request'
```

Comment:
One addition to this is to maybe add the RESOURCE_PATH to this tag so that we can uniquely differentiate between the different POST requests within the visual graphs. Maybe it could be something like `Post request - /fhir/Patient - Stress Test`